### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by mxcl** [teaxyz/teaBASE#24] [Announcements]
## What's Changed
* Adjust version number contrast by @stevenlei in https://github.com/teaxyz/teaBASE/pull/22
* Improve UI responsiveness during startup by @stevenlei in https://github.com/teaxyz/teaBASE/pull/21
* Fix: minor issue by @belalangeth in https://github.com/teaxyz/teaBASE/pull/23
* Improve Xcode and Command Line Tools Detection by @stevenlei in https://github.com/teaxyz/teaBASE/pull/17

## New Contributors
* @belalangeth made their first contribution in https://github.com/teaxyz/teaBASE/pull/23

**Full Changelog**: https://github.com/teaxyz/teaBASE/compare/v0.6.0...v0.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/teaxyz/teaBASE/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by raheestz** [comment]
oooi

**Comment by ajalaoluwasogo** [comment]
let see how it goes

**Comment by tanerxc** [comment]
lets see

**Comment by felixrobin256** [comment]
I'm going to see how it goers

**Comment by Onyxchuzzy** [comment]
I'm trying to figure out what to makecofvtgis as,a user, but let's see

**Comment by Heymma** [comment]
When windows?

**Reply by Dedorlar** [reply]
When window version gonna be released?

**Reply by mxcl** [reply]
Once we have this version nailed down we will consider other platforms.

**Comment by MztaJibola** [comment]
I wanted to download this on my windows laptop, and got to find out it's only available for MacOS. When are you planning to release a version for windows??

**Reply by mxcl** [reply]
Once we have this version nailed down we will consider other platforms.

**Comment by fahmisme** [comment]
Considering that TEA users are not only those who have the MacOs operating system, but many are also on Windows, maybe some are also Linux geeks. So, I hope you release it soon for other operating systems.

**Reply by mxcl** [reply]
Once we have this version nailed down we will consider other platforms.

**Comment by erameh6** [comment]
how do i go about it

**Reply by Gazamony** [reply]
which one

**Comment by Ayaan1982** [comment]
Good 👍
